ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/search_tree/algorithms/Leaf.java

```diff
@@ -478,10 +478,8 @@ public void excludeContingencies(Set<String> contingenciesToExclude) {
 
     @Override
     public Set<RangeAction<?>> getRangeActions() {
-        if (status == Status.EVALUATED) {
-            return prePerimeterActivationResult.getRangeActions();
-        } else if (status == Status.OPTIMIZED) {
-            return postOptimResult.getRangeActions();
+        if ((status == Status.EVALUATED) || (status == Status.OPTIMIZED)) {
+            return optimizationPerimeter.getRangeActions();
```

Comment:
I think you can completely remove the if/else.

Reply:
done